### PR TITLE
AWS 15.1.1

### PR DESCRIPTION
**Pull request by njuettner**

### CODE REVIEW

**Comment by njuettner** [comment]
Tested it manually in gauss and giraffe as running tests here lead to some issues, which is not related to the fix.

Good from my side to merge this.

**Review by calvix — APPROVED** [review]

**Review by anvddriesch — APPROVED** [review]

File: aws/v15.1.1/release.diff [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    name: metrics-server                                               name: metrics-server
+    version: 1.3.0                                                     version: 1.3.0
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.10.2                                                    version: 1.10.2
+  - componentVersion: 1.0.1                                          - componentVersion: 1.0.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.7.2                                                     version: 1.7.2
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    releaseOperatorDeploy: false                                <
```
Comment (alex-dabija) [inline aws/v15.1.1/release.diff:46]: `releaseOperatorDeploy`: false was removed, but the line is present for v15.1.0. Is false the default value?

Comment (njuettner) [reply aws/v15.1.1/release.diff]: thanks, fixed 👍🏻

**Pull request merged**